dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx

```diff
@@ -202,7 +205,7 @@ export const LogSheet = ({
           skeletonClassname="h-[3rem]"
           customError={
             <div className="p-4 text-center">
-              This log url is not supported in the log viewer yet
+              You can download your logs here
```

Comment:
I now this wasn't on a message but please put this on a message

Reply:
Completely forgot it, you are right